app/models/site.rb

```diff
@@ -7,10 +7,48 @@ class Site < ActiveRecord::Base
   accepts_nested_attributes_for :account, update_only: true
 
   class << self
-    delegate :account, :application_name, :institution_name, :institution_name_full, :reload, :update, to: :instance
+    delegate :account, :application_name, :institution_name,
+             :institution_name_full, :reload, :update, :announcement_text,
+             :marketing_text, :featured_researcher, to: :instance
 
     def instance
       first || create
     end
   end
+
+  def announcement_text
```

Comment:
@cbeer Correct, that model is provided by Sufia. Do you have something like the following in mind?

```ruby
class Site < ActiveRecord::Base
  # ...
  belongs_to :account
  has_many :content_blocks
  # ...
  class << self
    delegate :account, ..., :content_blocks, to: :instance
    delegate :announcement_text, :marketing_text, :featured_researcher, to: :content_blocks
    # ...
  end
end
```

If so, I'll need to create some convenience methods in Sufia (`#announcement_text`, `#marketing_text`, `#featured_researcher`) first.